<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateReport</class>
 <widget class="QDialog" name="CreateReport">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1" colspan="2">
      <widget class="QPlainTextEdit" name="reportQuery"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_reportName">
       <property name="text">
        <string>Report Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="hSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>228</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="label_warning">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>(Remember: With great power comes great responsibility!)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="reportName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_reportQuery">
       <property name="text">
        <string>Report Query</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>reportName</tabstop>
  <tabstop>reportQuery</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateReport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateReport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
